automation_tools/__init__.py

```diff
@@ -1071,6 +1071,15 @@ def product_install(distribution, create_vm=False, certificate_url=None,
             installer_options.update(execute(
                 setup_default_capsule, host=host, run_katello_installer=False
             )[host])
+        elif os.environ.get('INTERFACE'):
```

Comment:
This code can be written like this:

```python
          else:
              # execute returns a dictionary mapping host strings to the given
              # task's return value
             installer_options.update(execute(
                 setup_default_capsule,
                 host=host,
                 interface=os.environ.get('INTERFACE') or 'eth0',
                 run_katello_installer=False
             )[host])
```

I don't used os.environ.get('INTERFACE', 'eth0') because I want to handle `None` and `''` will default to `'eth0'`.